solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -131,7 +126,11 @@ private void extractTitles(Element mods, IndexDocumentBean idb) {
                 }
             }
         }
-        idb.setTitle(mainTitle);
+
+        if (mainTitle != null && !mainTitle.equals("")) {
```

Comment:
you can use StringUtils.isBlank in place of these two checks (the apache commons lang3 version)